common.py

```diff
@@ -104,18 +105,42 @@ def list_to_dict(input_list, key_fcn, value_fcn=lambda item: item, transform_fcn
     return dicted_list
 
 
-def setup():
+def get_access_token():
+    return GoogleCredentials.get_application_default().get_access_token().access_token
+
+# only needed until firecloud python library in pypi supports service accounts
+def _fiss_access_headers_local(headers=None):
+    """ Return request headers for fiss.
+        Retrieves an access token with the user's google crededentials, and
+        inserts FISS as the User-Agent.
+    Args:
+        headers (dict): Include additional headers as key-value pairs
+    """
     credentials = GoogleCredentials.get_application_default()
-    print "Using Google client id:", credentials.client_id
+    credentials = credentials.create_scoped(['https://www.googleapis.com/auth/userinfo.profile', 'https://www.googleapis.com/auth/userinfo.email'])
+    access_token = credentials.get_access_token().access_token
+    fiss_headers = {"Authorization" : "bearer " + access_token}
+    fiss_headers["User-Agent"] = firecloud_api.FISS_USER_AGENT
+    if headers:
+        fiss_headers.update(headers)
+    return fiss_headers
+
+
+def setup():
+    firecloud_api._fiss_access_headers = _fiss_access_headers_local
+    registration_info = requests.get("https://api.firecloud.org/register", headers=firecloud_api._fiss_access_headers())
+    if registration_info.status_code == 404:
+        fail("This account is not registered with FireCloud.")
+
+    print "Using credentials for firecloud account:", registration_info.json()["userInfo"]["userEmail"]
```

Comment:
we could, but this is mostly to give you a sanity check about what user you are running as since this can be really confusing when you get different results due to different application-default auth